usaspending_api/broker/management/commands/fpds_nightly_loader.py

```diff
@@ -272,10 +272,18 @@ def insert_new_fpds(self, to_insert, total_rows):
             # Append row to list of Awards updated
             AWARD_UPDATE_ID_LIST.append(award.id)
 
+            if row["last_modified"] and len(str(row["last_modified"])) == 19:
```

Comment:
Can you add a comment explaining the `19`? I'm assuming it's the length of `YYYY-MM-DD HH:MM:SS` but wanted to be sure.

Reply:
modified the line to use the actual length of the human-readable datetime format string and the comment says "19 characters"